src/types/sgid.ts

```diff
@@ -0,0 +1,19 @@
+export interface PublicOfficerData {
+  email: string
+  agencyName: string
+  departmentName: string
+  employmentTitle: string
+}
+
+export const isPublicOfficerData = (
+  item: unknown
+): item is PublicOfficerData => {
+  const officerData = item as PublicOfficerData
```

Comment:
dangerous assertion - we first need an assertion of `!!item` as this doesn't guarantee existence of `item`

Reply:
please do this before merge also

Reply:
https://github.com/isomerpages/isomercms-backend/pull/976/commits/eacf951ad2a268bfa36dcf10b25b42a2ff1956db